configure.ac

```diff
@@ -2766,6 +2766,44 @@ CheckFcitx()
     fi
 }
 
+dnl See if libdecor is available
+CheckLibDecor()
+{
+    AC_ARG_ENABLE(libdecor,
+[AS_HELP_STRING([--enable-libdecor], [use libdecor for Wayland client-side decorations [default=yes]])],
+                  , enable_libdecor=yes)
+    if test x$enable_libdecor = xyes; then
+        AC_MSG_CHECKING(for libdecor support)
+        AS_IF([$PKG_CONFIG --exists libdecor-0],
+              [video_libdecor=yes],
+              [video_libdecor=no])
+        AC_MSG_RESULT($video_libdecor)
+        if test x$video_libdecor = xyes; then
+            EXTRA_CFLAGS="$EXTRA_CFLAGS `$PKG_CONFIG --cflags libdecor-0`"
+            AC_DEFINE(HAVE_LIBDECOR_H, 1, [ ])
+
+            AC_ARG_ENABLE(libdecor-shared,
+[AS_HELP_STRING([--enable-libdecor-shared], [dynamically load libdecor [default=yes]])],
+                          , enable_libdecor_shared=yes)
+
+            dnl FIXME: ABI FREEZE! decor_lib=[`find_lib "libdecor-0.so.*" "" | sed 's/.*\/\(.*\)/\1/; q'`]
+            decor_lib=libdecor-0.so
```

Comment:
ABI's frozen, this can now be reverted.

Reply:
Ok, changed that to ``decor_lib=[`find_lib "libdecor-0.so.*" "" | sed 's/.*\/\(.*\)/\1/; q'`]``.